satysfi-fonts-dejavu-doc.opam

```diff
@@ -14,9 +14,9 @@ homepage: "https://github.com/zeptometer/SATySFi-fonts-dejavu"
 bug-reports: "https://github.com/zeptometer/SATySFi-fonts-dejavu/issues"
 dev-repo: "git+https://github.com/zeptometer/SATySFi-fonts-dejavu.git"
 depends: [
-  "satysfi" {>= "0.0.3" & < "0.0.4"}
+  "satysfi" {>= "0.0.3" & < "0.0.5"}
   "satyrographos" {>= "0.0.2" & < "0.0.3"}
-  "satysfi-fonts-dejavu" {= "2.37"}
+  "satysfi-fonts-dejavu" {>= "2.37"}
```

Comment:
Can you specify exactly the same version which correspond to the main library? In this case it should be `2.37+satysfi0.0.4`.

Users will be confused when they find satysfi-fonts-dejavu-doc does not match with satysfi-fonts-dejavu.